lib/config.js

```diff
@@ -2,8 +2,11 @@
  * Copyright (c) 2015-2022 Digital Bazaar, Inc. All rights reserved.
  */
 import {config} from '@bedrock/web';
+import {vcViewConfigurations} from './vcViewConfigurations.js';
 
 const cfg = {
+  // Customizable VC configurations
+  vcViewConfigurations,
```

Comment:
So using the name `configurations` inside of the config is redundant now -- we can shorten it. Will take this to another medium for discussion.

Reply:
Rename vcViewConfigurations to cardDesigns - 7b2911f